usl_pipeline/cloud_functions/requirements.txt

```diff
@@ -35,7 +35,7 @@ itsdangerous==2.1.2
 Jinja2==3.1.3
 MarkupSafe==2.1.5
 netCDF4==1.6.5
-numpy==1.26.4
+numpy>=1.25.2
```

Comment:
@waltaskew Do you know a better way to resolve numpy install issue that's failing the build?

I changed this line because of subprocess error when installing `wrf-python` - this lib will also install numpy as a dep but expects it to be a a different version. pip install runs fine when I install the requirements in a local env but something in the github action is not happy..

Reply:
we should set the right numpy version in here, then. We want these to be exact `==` versions to avoid unexpected version changes in deployment.

The process for updating this file should be
```python
pip install <all the things>
pip freeze > requirements.txt
```
so you'll export whatever versions `pip` resolved the various libraries to

Reply:
failing how? When I pip install wrf-python and the other things in the requirements file, pip selects `numpy==1.25.2`

Reply:
Tried setting it to 1.15.2 too but:
https://github.com/UrbanSystemsLab/climateiq-cnn/actions/runs/9764950697/job/26954503839?pr=97

Reply:
This should be fixed once https://github.com/UrbanSystemsLab/climateiq-cnn/pull/108 is merged